src/shared/lib/helpers/getDateAndTime/getDateAndTime.test.ts

```diff
@@ -0,0 +1,11 @@
+import { getDateAndTime } from './getDateAndTime';
+
+describe('getDateAndTime', () => {
+    test('with time', () => {
+        expect(getDateAndTime(1699860692936))
+            .toEqual('13.11.2023 / 10:31:32');
```

Comment:
this test fails for me as local settings on my machine not same as on yours
```
    Expected: "13.11.2023 / 10:31:32"
    Received: "11/13/2023 / 8:31:32 AM"
```
You can just update the test a bit but as optional improvement I would suggest to use https://momentjs.com/ it's much more flexible